test/TesterInternal/MembershipTests/MembershipTableTestsBase.cs

```diff
@@ -102,43 +103,56 @@ protected virtual string GetAdoInvariant()
             Assert.True(entries.Contains(membershipEntries[9].SiloAddress.ToGatewayUri().ToString()));
         }
 
-        protected async Task MembershipTable_ReadAll_EmptyTable()
+        protected async Task MembershipTable_ReadAll_EmptyTable(bool extendedProtocol = true)
         {
             var data = await membershipTable.ReadAll();
             Assert.NotNull(data);
 
-            logger.Info("Membership.ReadAll returned VableVersion={0} Data={1}", data.Version, data);
+            if (extendedProtocol)
```

Comment:
You are right. While rerunning the tests, only the ones that increment and validates the increment of the tableVersion row fail. I'm re-enabling those tests and leaving the flag only on those who fail.